### DISCUSSION TITLE
Neolink on Windows 11

### DISCUSSION THREAD

**Discussion by IssaPontiacG8** [thirtythreeforty/neolink#259] [Q&A]
Greetings all. Upgrading my BlueIris setup from an i5 6500 to an i7 8700 and going to Windows 11. I was able to get BlueIris set up, all my stuff copied over without issue. Copied over my neolink files/settings without issue. Reinstalled gstreamer without issue. But, I cannot access the rtsp streams. I don't see any errors and everything looks fine. I open wireshark, and see tons of traffic between my 4 cameras. I can copy/paste the files into the old system and everything works fine without changing anything. BlueIris pointed to the old system works fine, so definitely something with the new Windows 11 system and rtsp stream not coming up correctly. Anyone have thoughts?

![Logs](https://user-images.githubusercontent.com/99851956/159139664-eb8a85f1-0fd9-4653-b5d4-ae34341dd141.PNG)

**Comment by QuantumEntangledAndy** [comment]
I think we should try some general debugging.

Things to try:
- Put only one camera in the config
- Connect to the stream via an ffplay that is running on the same machine as neolink
- Set the environmental variable `GST_DEBUG` to 3 then rerun neolink and try to connect to it.
  - When an rtsp connection is started it will give details on what the rtsp client is trying to do and this will tell us what is wrong. 
  - This mode prints a lot of stuff though so it can be overwhelming to process. Post it here for help

**Reply by workstations-ac** [reply]
I keep getting gstreamer errors.  I've check the environment path and yet neolink will not run.  Has anyone run into this at all?
